CHANGELOG.md

```diff
@@ -128,6 +128,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
   - Addition of Doc values on the GeoShape Field
   - Addition of GeoShape ValueSource level code interfaces for accessing the DocValues.
   - Addition of Missing Value feature in the GeoShape Aggregations.
+- Added feature to ignore indexes starting with dot during shard limit validation.([#4674](https://github.com/opensearch-project/OpenSearch/pull/4674))
```

Comment:
nit: indentation?

Reply:
The above comments to this one are sub comments however the space is same as how parent comments are defined.